### ISSUE TITLE
Can the 'Groups' listing be sorted by popularity?

### ISSUE THREAD

**Issue by saisiddu** [hydroshare/hydroshare#3322]
Hi, I am a developer of 'Landlab' (landlab.github.io). We have an active user group on Hydroshare. I was wondering if you can list the Hydroshare groups by popularity? Thanks!

**Comment by cuahsimarko** [comment]
Hi, Sai, thanks for reaching out! We're working on some bugfixes right now, but I will be sure this gets attention when we can get to it. In the meantime, I'm interested in your project and anything we can do to help interacting with the HydroShare codebase. For example, we are beta testing a new developer setup and some of your more technically advanced team mates may want to try it.

**Comment by saisiddu** [comment]
Hi Mark, Thank you for your message. Christina Bandaragoda and Tony Castranova are working with us regarding beta-testing. I wasn't able to tag them. They also asked me to tag them to this issue but I don't have permission to do so. Can you please tag them for me?

**Comment by cuahsimarko** [comment]
Yes, I work with Tony here at the office. I learned after reaching out to our team about this that you have already been connected. I will be sure to include Christina and Tony in the progress and design of this, as we discuss with you.

**Comment by cuahsimarko** [comment]
being discussed via e-mails... still underway

**Comment by cuahsimarko** [comment]
@Lizabrazil will lead the design discussion per e-mail thread just forwarded. Developers will re-engage when more is locked down.

**Comment by cuahsimarko** [comment]
@Lizabrazil let's keep the thread alive here

**Comment by Lizabrazil** [comment]
Christina and Sai proposed a popularity index to define Popularity in groups:

 # members/total HS users 
+ 
# accessed resources this week/ # of all accessed resources on Hydroshare 
+ 
# new members in the past 12 months/total new HS members in the past 12 months
-----------
It will be a weighted average of:
1) total members 
2) recent members (no. of people that joined in the last month or three months)
3) active members in the last month or three months - because people may join and use Hydroshare for a week but never come back
-----------
Discussion needs to take place with @Castronova to figure out what is a feasible metric to maintain here. I have an associated JIRA ticket here: https://cuahsi.atlassian.net/browse/DEV-67

**Comment by Lizabrazil** [comment]
@cuahsimarko if we can generate a popularity index, do you know how technically feasible it would be to automatically move around the order of the groups? This seems a little more involved, do you have the bandwidth to look into this?

**Comment by ClaraCogswell** [comment]
Closing: not super feasible, low impact, much work ongoing on communities and groups, may be addressed or quickly become defunct giving ongoing work on communities.
